lib/commands/appearance.js

```diff
@@ -0,0 +1,55 @@
+import _ from 'lodash';
+
+
+const commands = {};
+
+/**
+ * @typedef {Object} SetAppearanceOptions
+ *
+ * @property {string} style - Currently two styles are supported:
+ * - dark
+ * - light
+ */
+
+/**
+ * Set the device's UI appearance style
+ *
+ * @since Xcode SDK 11.4
+ * @param {SetAppearanceOptions} opts
+ */
+commands.mobileSetAppearance = async function mobileSetAppearance (opts = {}) {
+  const {
+    style,
+  } = opts;
+  if (!style) {
+    throw new Error(`The 'style' option is expected to be present`);
+  }
+
+  return this.isSimulator()
+    ? await this.opts.device.setAppearance(style)
+    // ugly workaround for real devices
+    : await this.mobileSiriCommand({
```

Comment:
Can the `mobileSiriCommand` workaround be used on simulators with Xcode < 11.4 ?

Reply:
yes, we can try it